.github/workflows/security.yml

```
@@ -263,50 +449,473 @@ jobs:
+          echo "$CODE_VULNS_DETAILS" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+          echo "ENV_FILES_DETAILS<<EOF" >> $GITHUB_ENV
+          echo "$ENV_FILES_DETAILS" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+          echo "✅ Extraction des détails terminée"
+
       - name: 🚨 Create critical security issue
-        if: (needs.security-deep-scan.outputs.critical-vulnerabilities > 0 || needs.dependency-check.outputs.dependency-vulnerabilities > 0) && (github.event.inputs.create_issue == 'true' || github.event.inputs.create_issue == '')
+        if: (needs.security-deep-scan.outputs.critical-vulnerabilities > 0 || needs.dependency-check.outputs.critical-vuln-count > 0 || needs.security-deep-scan.outputs.secrets-found > 0) && (github.event.inputs.create_issue == 'true' || github.event.inputs.create_issue == '')
```

actionlint (1.7.7)

property "critical-vuln-count" is not defined in object type {dependency-vulnerabilities: string}

(expression)